### PR TITLE
added is_available method in Game model and added available column

### PR DESCRIPTION
**Pull request by FsDevNinja**

### CODE REVIEW

**Comment by kellylougheed** [comment]
Looks good to me. Is it still in a work in progress? Do we need tests for this method?

**Comment by FsDevNinja** [comment]
Yeah. I'm still working on the tests for this one. Not sure what to do.

**Comment by kellylougheed** [comment]
Is the test passing @FsDevNinja? Trying to figure out why the build is failing.

**Comment by FsDevNinja** [comment]
yeah, when i took out the available column, it broke some tests. its fixed.

File: app/models/game.rb [hunk 1]
```diff
@@ -1,6 +1,11 @@
 class Game < ActiveRecord::Base
-
+  
+  scope :is_available, -> { where("black_player_id is null or white_player_id is null") }
+  
+
```
Comment (lasley) [inline app/models/game.rb]: Why so many carriage returns? Fall asleep on the keyboard? 😛

File: db/schema.rb [hunk 2]
```diff
@@ -11,10 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-
-
-ActiveRecord::Schema.define(version: 20161115154942) do
-  
+ActiveRecord::Schema.define(version: 20161118012333) do
```
Comment (lasley) [inline db/schema.rb]: Should completely revert the changes on this file. Something like:
```
git checkout master db/schema.rb
```

Comment (FsDevNinja) [inline db/schema.rb]: Is that a functional change or OCD? 😀

Comment (lasley) [inline db/schema.rb]: `shema.rb` should never be edited by hand

**Comment by kellylougheed** [comment]
Ready to approve this, but do we need to revert the changes on schema.rb like @lasley suggested, @FsDevNinja?

**Comment by kellylougheed** [comment]
Ack, it looks like build failed. Do we need to change the schema back again?

Another issue: in my branch, if the player who creates the game chooses to be black, the white_player_id ends up as 0 (I've tried to make it nil with no luck). Any way that is_available could cover the possibility of white_player_id being null or 0?

**Comment by kellylougheed** [comment]
Thanks @FsDevNinja!
